governance/votes/Executive vote - November 2, 2022.md

```diff
@@ -0,0 +1,96 @@
+
+Please review [MIP39](https://mips.makerdao.com/mips/details/MIP39) for details on offboarding Core Units and the forum threads discussing the [offboarding of the Events Core Unit (EVENTS-001)](https://forum.makerdao.com/t/mip39c3-sp4-offboarding-the-events-core-unit-events-001/17755), the [offboarding of the Strategic Happiness Core Unit (SH-001)](https://forum.makerdao.com/t/mip39c3-sp3-offboarding-the-strategic-happiness-core-unit-sh-001/17754), and the [offboarding of the Real-World Finance Core Unit (RWF-001)](https://forum.makerdao.com/t/mip39c3-sp5-offboarding-the-real-world-finance-core-unit-rwf-001/17753) to inform your position on this issue before voting.
+
+
+
+
+### Offboarded Core Units - DAI and MKR Golden Parachute Budget Transfers
+
+As per multiple successful offboarding proposals (linked below), a total of **210,998 DAI** and **169.5 MKR** will be transferred to multiple offboarded Core Units as part of their respective Golden Parachutes if this executive proposal passes.
+* **167,666 DAI** will be transferred to the [Events Core Unit (EVENTS-001)](https://mips.makerdao.com/mips/details/MIP39c2SP32) at 0x3D274fbAc29C92D2F624483495C0113B44dBE7d2 as a result of a successful [offboarding proposal](https://mips.makerdao.com/mips/details/MIP39c3SP4).
+* **43,332 DAI** will be transferred to the [Strategic Happiness Core Unit (SH-001)](https://mips.makerdao.com/mips/details/MIP39c2SP11) at 0xc657aC882Fb2D6CcF521801da39e910F8519508d as a result of a successful [offboarding proposal](https://mips.makerdao.com/mips/details/MIP39c3SP3).
+* **26.04 MKR** will be transferred to the [Strategic Happiness Core Unit (SH-001)](https://mips.makerdao.com/mips/details/MIP39c2SP11) at 0xc657aC882Fb2D6CcF521801da39e910F8519508d as a result of a successful [offboarding proposal](https://mips.makerdao.com/mips/details/MIP39c3SP3).
+* **143.46 MKR** will be transferred to the [Real-World Finance Core Unit (RWF-001)](https://mips.makerdao.com/mips/details/MIP39c2SP1) at 0xD1505ee500791490DE8642353BA6A5b92e3550F7 as a result of a successful [offboarding proposal](https://mips.makerdao.com/mips/details/MIP39c3SP5).
+
+Please review [MIP39](https://mips.makerdao.com/mips/details/MIP39) for details on offboarding Core Units and the forum threads discussing the [offboarding of the Events Core Unit](https://forum.makerdao.com/t/mip39c3-sp4-offboarding-the-events-core-unit-events-001/17755), the [offboarding of the Strategic Happiness Core Unit](https://forum.makerdao.com/t/mip39c3-sp3-offboarding-the-strategic-happiness-core-unit-sh-001/17754), and the [offboarding of the Real-World Finance Core Unit](https://forum.makerdao.com/t/mip39c3-sp5-offboarding-the-real-world-finance-core-unit-rwf-001/17753) to inform your position on this issue before voting.
```

Comment:
add link to my summary thread - https://forum.makerdao.com/t/executive-vote-cu-offboarding-next-steps/18522